src/PresentationalComponents/Cards/ResolutionCard.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/prop-types */
+import './Pathways.scss';
+
+import {
+  Card,
+  CardBody,
+  CardTitle,
+} from '@patternfly/react-core/dist/esm/components/Card/index';
+import { InsightsLabel } from '@redhat-cloud-services/frontend-components/InsightsLabel';
+import { RISK_OF_CHANGE_LABEL } from '../../AppConstants';
+import React from 'react';
+import { RebootRequired } from '../Common/Common';
+import RecommendationLevel from '../Labels/RecommendationLevel';
+import messages from '../../Messages';
+import { useIntl } from 'react-intl';
+
+export const ResolutionCard = (props) => {
```

Comment:
Same, same, but different. :D

Reply:
For this, props are passed down as an argument for Recommendation lvl, do you think we should change recommendationLevel to just accept lvl and refactor things around it? (no isCompact just because all of them are gonna be compact)

Reply:
> do you think we should change recommendationLevel to just accept lvl and refactor things around it?

Yes. :)